### ISSUE TITLE
Reconnect HB to deployments for google-books app

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/google-books#347]
It looks like something changed on 2/21
https://app.honeybadger.io/projects/67990/deploys

Seems to be associated with this error on deploy:
```
01:30 honeybadger:deploy
      Notifying Honeybadger of deploy.
      01 RAILS_ENV=production bundle exec honeybadger deploy --environment stage --revision 0190127eabcf48686194c5e301abe3b1bf59c…
      01 bash: bundle: command not found
```

**Comment by jcoyne** [comment]
I've noticed that other capistrano tasks for google-books run:
```
/usr/local/rvm/bin/rvm default do bundle exec rake ...
```

but the HB one just runs:
```
RAILS_ENV=production bundle exec honeybadger ...
```

**Comment by jcoyne** [comment]
Fixed by #370
